src/core.js

```diff
@@ -606,12 +606,16 @@ $.extend( $.validator, {
 			return $( this.settings.errorElement + "." + errorClass, this.errorContext );
 		},
 
-		reset: function() {
+		resetLists: function() {
```

Comment:
I think it's a bit confused for someone who started reading the code for the first time as it's the only method that has the name started with an `_`